app-agent/Cargo.toml

```diff
@@ -24,6 +24,7 @@ plugin-rapl = { path = "../plugin-rapl", optional = true }
 plugin-socket-control = { path = "../plugin-socket-control", optional = true }
 plugin-relay = { path = "../plugin-relay", optional = true }
 plugin-procfs = { path = "../plugin-procfs", optional = true }
+plugin-energy-estimation-tdp = { path = "../plugin-energy-estimation-tdp", optional = true }
```

Comment:
I don't think that this plugin should be in the default agents

Reply:
That could be discussed indeed! The impact on the size of the binary would need to be evaluated (in particular, each plugin can have different dependencies). Multiple agent flavors could be proposed with a nice interface. I've got some ideas on a sheet somewhere :)